src/main/webapp/views/home.html

```diff
@@ -25,28 +25,31 @@
          <!-- Modal -->
         <div id="registration-modal"></div>
         <div class="row align-items-center justify-content-center mt-5">
-          <h1 class="home-header mt-3">Welcome to Vendr</h1>
+          <h1 class="home-header mt-3 display-2">Hello Vendr</h1>
         </div>
         <div id="tutorial-container" class="container h-100">
           <div class="row h-75 align-items-center">
             <div class="col">
               <div id="carousel-tutorial" class="carousel slide" data-interval="false">
-                <ol class="carousel-indicators">
-                  <li data-target="#carousel-tutorial" data-slide-to="0" class="active"></li>
-                  <li data-target="#carousel-tutorial" data-slide-to="1"></li>
-                  <li data-target="#carousel-tutorial" data-slide-to="2"></li>
-                </ol>
                 <div class="row h-100 align-items-center justify-content-center">
-                  <div id="tutorial-content" class="carousel-inner container"></div>
+                  <div id="tutorial-content" class="carousel-inner container h-100"></div>
                 </div>
-                <a class="carousel-control-prev" href="#carousel-tutorial" role="button" data-slide="prev">
+                <a class="carousel-control-prev fix-width" href="#carousel-tutorial" role="button" data-slide="prev">
                   <span class="carousel-control-prev-icon" aria-hidden="true"></span>
-                  <span class="sr-only">Previous</span>
+                  <span class="sr-only">Previous Slide</span>
                 </a>
-                <a class="carousel-control-next" href="#carousel-tutorial" role="button" data-slide="next">
+                <a class="carousel-control-next fix-width" href="#carousel-tutorial" role="button" data-slide="next">
                   <span class="carousel-control-next-icon" aria-hidden="true"></span>
-                  <span class="sr-only">Next</span>
+                  <span class="sr-only">Next Slide</span>
                 </a>
+                <ol class="carousel-indicators">
+                  <li title="Slide 1" data-target="#carousel-tutorial" data-slide-to="0" class="active"></li>
+                  <li title="Slide 2" data-target="#carousel-tutorial" data-slide-to="1"></li>
+                  <li title="Slide 3" data-target="#carousel-tutorial" data-slide-to="2"></li>
+                  <li title="Slide 4" data-target="#carousel-tutorial" data-slide-to="3"></li>
+                  <li title="Slide 5" data-target="#carousel-tutorial" data-slide-to="4"></li>
+                  <li title="Slide 6" data-target="#carousel-tutorial" data-slide-to="5"></li>
+                </ol>
```

Comment:
Can the title of this markers be more descriptive? for example: how to register, how to submit you business info

Reply:
It required for two different types of indicator html files but It is already done!